Dolphin scripts/Entrance Randomizer/lib/utils.py

```diff
@@ -50,21 +50,30 @@ def dump_spoiler_logs(
     seed_string: SeedType,
 ):
     spoiler_logs = f"Starting area: {starting_area_name}\n"
+    red_string_list: list[str] = []
     for original, redirect in transitions_map.items():
-        spoiler_logs += (
+        red_string_list.append(
             f"{TRANSITION_INFOS_DICT[original[0]].name} "
             + f"({TRANSITION_INFOS_DICT[original[1]].name} exit) "
             + f"will redirect to: {TRANSITION_INFOS_DICT[redirect[1]].name} "
-            + f"({TRANSITION_INFOS_DICT[redirect[0]].name} entrance)\n"
+            + f"({TRANSITION_INFOS_DICT[redirect[0]].name} entrance)\n",
         )
+    red_string_list.sort()
+    for string in red_string_list:
+        spoiler_logs += string
 
     unrandomized_transitions = ALL_POSSIBLE_TRANSITIONS - transitions_map.keys()
-    spoiler_logs += "\nUnrandomized transitions:\n"
-    for transition in unrandomized_transitions:
-        spoiler_logs += (
-            f"From: {TRANSITION_INFOS_DICT[transition[0]].name}, "
-            + f"To: {TRANSITION_INFOS_DICT[transition[1]].name}.\n"
-        )
+    if len(unrandomized_transitions) > 0:
+        spoiler_logs += "\nUnrandomized transitions:\n"
+        non_random_string_list: list[str] = []
+        for transition in unrandomized_transitions:
+            non_random_string_list.append(
+                f"From: {TRANSITION_INFOS_DICT[transition[0]].name}, "
+                + f"To: {TRANSITION_INFOS_DICT[transition[1]].name}.\n",
+            )
```

Comment:
The linter here is asking for 
```suggestion
        non_random_string_list = [
            (
                f"From: {TRANSITION_INFOS_DICT[transition[0]].name}, "
                + f"To: {TRANSITION_INFOS_DICT[transition[1]].name}.\n"
            )
            for transition in unrandomized_transitions
        ]
```

Reply:
Done.